test/stability/testSingleWorkingFF/testSingleWorkingFF.go

```diff
@@ -256,8 +255,7 @@ func executeTest(configFile, target string, testScenario uint, testType uint) er
 }
 
 func setParameters(testScenario uint) (err error) {
-	eps = 3
-	passedLimit = 85
+	passedLimit = 98
```

Comment:
Can you use a named constant defined at the beginning of the file instead of magic number 98?

Reply:
It is just name inside setParameters